Chapter12/recipe-02/fortran-example/account/implementation/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+add_library(
+  account
+  SHARED
+  fortran_implementation.f90
+  )
+
+# generate account_export.h
+include(GenerateExportHeader)
+generate_export_header(
+  account
+  BASE_NAME account
+  )
+
+install(
+  FILES
+    ${PROJECT_BINARY_DIR}/account/implementation/account_export.h
+  DESTINATION
+    include
+  )
+
+install(
+  TARGETS
+    account
+  LIBRARY DESTINATION lib
```

Comment:
Ditto about `GNUInstallDirs.cmake`. On Windows this might want to be modified to:
```
install(
  TARGETS
     account
  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
  RUNTIME DESTINATION ${CMAKE_INSTALL_LIBDIR}
  )
```